### PR TITLE
[Feature] Allows admin to edit existing debates

### PR DESCRIPTION
**Pull request by Sqyphen**
## Description

Allows admin to edit existing debates

## Implementation

Adds new edit debate api query and corrected DynamoDB calls

## Screenshot

n/a

## .env changes

n/a

## Additional requirements

n/a

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,6 +34,10 @@ app.engine(
   })
 );
 
+hbs.registerHelper("ifEquals", function(arg1, arg2, options) {
```
Comment (Lily2point0) [inline index.js]: Can we have the helpers in a separate utils file?

File: models/dynamo_db.js [hunk 2]
```diff
@@ -141,14 +182,22 @@ async function query(type, params) {
...
 
-    if (type === "get") {
-      result = await dynamoDb.get(allParams).promise();
-    } else if (type === "scan") {
-      result = await dynamoDb.scan(allParams).promise();
-    } else if (type === "put") {
-      result = await dynamoDb.put(allParams).promise();
-    } else if (type === "query") {
-      result = await dynamoDb.query(allParams).promise();
+    switch (type) {
```
Comment (Lily2point0) [inline models/dynamo_db.js]: no default?

Comment (Sqyphen) [reply models/dynamo_db.js]: added ⬆️

File: index.js (hunk 1 above)
Comment (Sqyphen) [reply index.js]: Done

**Pull request merged**